k8s-deploy/pkg/service/kube_namespace.go

```diff
@@ -16,14 +16,15 @@
 package service
 
 import (
+	"github.com/pkg/errors"
 	v1 "k8s.io/api/core/v1"
 )
 
 // GetNamespaces GetNamespaces
 func GetNamespaces() ([]v1.Namespace, error) {
 	namespaceList, err := KubeClient.GetNamespaces()
 	if err != nil {
-		return nil, err
```

Comment:
We'd better use `errors.Wrap()` function to keep the original error in the new error.